lib/modals/actions_dialog.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+import 'package:regatta_buddy/modals/action_button.dart';
+import 'package:regatta_buddy/utils/constants.dart' as constants;
+
+Future<void> showActionsDialog(
+    BuildContext context, List<ActionButton> actions) async {
+  await Future.delayed(const Duration(seconds: 0));
+  if (context.mounted) {
+    showDialog(
+      barrierDismissible: true,
+      context: context,
+      builder: (BuildContext cxt) {
+        var autoClosingActions = actions.map((action) {
+          action.setAdditionalCallback(() => Navigator.of(cxt).pop(false));
```

Comment:
how to understand pop(false)?

Reply:
It is just for clarity and the value is returned to the screen 'below'. I hope that makes sense XD